mathesar_ui/src/components/NameAndDescInputModalForm.svelte

```diff
@@ -18,6 +22,8 @@
   import { toast } from '@mathesar/stores/toast';
   import TextArea from '@mathesar/component-library/text-area/TextArea.svelte';
 
+  export let helpText = '';
```

Comment:
`helpText` would work better as a slot than as a prop. I see in the parent component you're passing a huge string into this prop. If you use a slot, you'll be able to move that text out of a string and put it directly into the template. This also makes it possible to put formatting in the help text.

Reply:
Create a good first here: #1858